src/Components/NavBar.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styled from 'styled-components';
+
+
+
+const NavBar = () => {
+	return (
+		<Nav>
+			<img>
+			</img>
+			<div>
+				<Button>
+					<h3>All Crafts</h3>
+				</Button>
+				<Button>
+					<h3>Profile</h3>
+				</Button>
+			</div>
+		</Nav>
+	)
+}
+
```

Comment:
I feel like indentation here is a little extreme. Super small detail, but we might be able to clean this up a bit.